### PR TITLE
[WebApp] Generate default passkey name

### PR DESCRIPTION
**Pull request by vitto-moz**
https://www.notion.so/saladtech/FE-Add-default-passkey-name-generating-functionality-84f0c3e25da14816aa2dcb69b08ca809?pvs=4

### CODE REVIEW

File: packages/web-app/src/modules/passkey-setup/components/utils.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const getBrowserName = () => {
+  const opera = navigator.userAgent.indexOf('OPR') !== -1
+  console.log('opera ===> ', opera)
```
Comment (Maks19) [inline packages/web-app/src/modules/passkey-setup/components/utils.ts:3]: I think we can remove it now

File: packages/web-app/src/modules/passkey-setup/components/utils.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  if (navigator.userAgent.indexOf('Opera') !== -1 || navigator.userAgent.indexOf('OPR') !== -1) {
+    console.log('test ===> ')
```
Comment (Maks19) [inline packages/web-app/src/modules/passkey-setup/components/utils.ts:5]: I think we can remove it now

File: packages/web-app/src/modules/passkey-setup/components/utils.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
+const getBrowserName = () => {
+  const opera = navigator.userAgent.indexOf('OPR') !== -1
```
Comment (Maks19) [inline packages/web-app/src/modules/passkey-setup/components/utils.ts:2]: Why do you declare opera here, and not in if function?

File: packages/web-app/src/modules/passkey-setup/components/utils.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  console.log('opera ===> ', opera)
+  if (navigator.userAgent.indexOf('Opera') !== -1 || navigator.userAgent.indexOf('OPR') !== -1) {
+    console.log('test ===> ')
+    return 'Opera'
+    // @ts-ignore
+  } else if (navigator.userAgent.indexOf('Edg') !== -1) {
```
Comment (Maks19) [inline packages/web-app/src/modules/passkey-setup/components/utils.ts:8]: It looks like switch case construction. Why do you use if else?

File: packages/web-app/src/modules/passkey-setup/components/utils.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  } else if (navigator.userAgent.indexOf('Safari') !== -1) {
+    return 'Safari'
+  } else if (navigator.userAgent.indexOf('Firefox') !== -1) {
+    return 'Firefox'
+    // @ts-ignore
+  } else if (navigator.userAgent.indexOf('MSIE') !== -1 || !!document.documentMode === true) {
+    //IF IE > 10
+    return 'IE'
+  } else {
+    return 'Unknown'
```
Comment (Maks19) [inline packages/web-app/src/modules/passkey-setup/components/utils.ts:21]: What do you think about using more meaningful default names if we can't get browser and os?
For example, 'Default Passkey', as we had previously.

**Review by Maks19 — CHANGES_REQUESTED** [review]

File: packages/web-app/src/modules/passkey-setup/components/utils.ts (hunk 4 above)
Comment (vitto-moz) [reply packages/web-app/src/modules/passkey-setup/components/utils.ts:8]: Could you suggest desired refactor code?

File: packages/web-app/src/modules/passkey-setup/components/utils.ts (hunk 5 above)
Comment (vitto-moz) [reply packages/web-app/src/modules/passkey-setup/components/utils.ts:21]: changed to "Unknown Browser"

File: packages/web-app/src/modules/passkey-setup/components/utils.ts (hunk 3 above)
Comment (vitto-moz) [reply packages/web-app/src/modules/passkey-setup/components/utils.ts]: this line is not necessary at all 
removed

File: packages/web-app/src/modules/passkey-setup/components/utils.ts (hunk 4 above)
Comment (Maks19) [reply packages/web-app/src/modules/passkey-setup/components/utils.ts:5]:
```
switch (true) {
  case userAgent.indexOf('Opera') !== -1 || userAgent.indexOf('OPR') !== -1:
    return 'Opera';

  case userAgent.indexOf('Edg') !== -1:
    return 'Edge';

  case userAgent.indexOf('Chrome') !== -1:
    return 'Chrome';

  case userAgent.indexOf('Safari') !== -1:
    return 'Safari';

  case userAgent.indexOf('Firefox') !== -1:
    return 'Firefox';

  case userAgent.indexOf('MSIE') !== -1 || !!document.documentMode === true:
    return 'IE';

  default:
    return 'Unknown Browser';
}
```

Comment (Maks19) [reply packages/web-app/src/modules/passkey-setup/components/utils.ts:5]: Any reason to use if else instead of switch(true)?

Comment (vitto-moz) [reply packages/web-app/src/modules/passkey-setup/components/utils.ts]: refactored to switch

**Review by Maks19 — APPROVED** [review]

**Pull request merged**